src/Waypoint/WaypointReaderSeeYou.hpp

```diff
@@ -44,6 +44,11 @@ class WaypointReaderSeeYou final : public WaypointReaderBase {
 protected:
   /* virtual methods from class WaypointReaderBase */
   bool ParseLine(const TCHAR* line, Waypoints &way_points) override;
+
+private:
+  /* field positions for typical SeeYou *.cup waypoint file */
+  unsigned short iFrequency = 9;
```

Comment:
Why is this a `short`? Are you aware that `short` is slower than a default-sized integer? Sure, your way, two `shorts` can be packed into one 32 bit word, saving 4 bytes of memory (theoretically, but only on a 32 bit CPU which is a fading technology). But the saving win must outweigh the performance loss, for example when you have thousands of instances; but this class has only temporary instances, and usually only one at a time, during startup only.